cs/src/Contracts/TunnelService.Contracts.csproj

```diff
@@ -1,8 +1,8 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <RootNamespace>Microsoft.VsSaaS.TunnelService.Contracts</RootNamespace>
-    <AssemblyName>Microsoft.VsSaaS.TunnelService.Contracts</AssemblyName>
+    <RootNamespace>Microsoft.DevTunnels.TunnelService.Contracts</RootNamespace>
```

Comment:
Should the namespace be just `Microsoft.DevTunnels.Contracts`? The `TunnelService` part seems redundant, and maybe a little confusing since this is not service code.

Same for the Management and Connections project.

Reply:
Updated.